### ISSUE TITLE
Statistics

### ISSUE THREAD

**Issue by Jackster35** [Eureka22/agar-clone#29]
With vram4's latest push, now showing statistics on another port, would there be a way of getting statistics for multiple servers and showing them on the server select?

https://github.com/forairan/Ogar/commit/e3330b667f0fb26a877a466ea7c0de29a7605fe9

If you'd like a link to my server's stats page, feel free to email me at me@eptic.me and I'll send you a link. Just as something for you to work off maybe? :)

**Comment by m-byte** [comment]
I think, this issue belongs in Cigar ;)
Also, what kind of statistics do you want to show? The users connected per server? If so, that should be possible in the master server.

**Comment by Jackster35** [comment]
Yeah, users per server. And I used the fork I posted, so master server isn't in that. (I don't really wanna be switching forks and server files tbh) haha, I've done quite a lot of that recently and I have loads of traffic so server restarting is always the last thing I want to do.

**Comment by m-byte** [comment]
That's a feature that is already implemented in the client when using the master server, though. It's a lot more complicated to load those stats from each individual server. Also, there's a lot more traffic than needed.

**Comment by Jackster35** [comment]
I guess, but it's possible without the huge amount of traffic to the game servers.

**Comment by vram4** [comment]
You could just go to the /info page for the player count, assuming you have 1 server per region.
